docs/tutorials/basic_ua_commands.md

```diff
@@ -0,0 +1,258 @@
+# Tutorial: cover base UA commands
+
+The Ubuntu Advantage (UA) Tools provides users with a simple mechanism to
```

Comment:
The rationale here for tools is that the package is called `ubuntu-advantage-tools`, so that's why I named it like this. But we can change it to Client in the docs if we all agree

Reply:
We refer to it as client pretty much everywhere except for the package name I think so far haha :upside_down_face: